inst/tutorials/data_visualization_basic/data_visualization_basic.Rmd

```diff
@@ -0,0 +1,301 @@
+---
+title: "Introduction to data visualization"
+author: "Julia Anstett and Dr. Stephan Koenig (adpated from Dr. Kim Dill-McFarland)"
+date: "version `r format(Sys.time(), '%B %d, %Y')`"
+output:
+  learnr::tutorial:
+    progressive: true
+    allow_skip: true
+runtime: shiny_prerendered
+description: Basic data visualization with ggplot2 focusing on dot plots.
+---
+
+```{r setup, include = FALSE}
+# General learnr setup
+library(learnr)
+knitr::opts_chunk$set(echo = TRUE)
+library(educer)
+# Helper function to set path to images to "/images" etc.
+setup_resources()
+
+library(dplyr)
+library(readr)
+
+raw_dat <- geochemicals
+
+dat <- mutate(raw_dat, Depth = as.factor(Depth),
+              Cruise = as.factor(Cruise))
+
+dat <- filter(dat, Depth %in% c(10, 100, 200),
+              # The cruises that took place in Februrary can be determined
+              # using the Date variable and functions that we won't discuss in
+              # these data science modules
+              Cruise %in% c(18, 30, 42, 54, 66, 80, 92))
+```
+
+## Introduction
+This tutorial provides guided, hands-on instruction to create and modify plots using the `ggplot2` package in R. It also includes plots of an ANOVA analysis.
+
+After this tutorial, you will be able to:
+
+* Create dot and box plots in `ggplot2`
+* Modify attributes of ggplots
+* Complete and interpret the output of ANOVAs in R
+
+## Setup
+Prior to starting this tutorial, please complete the *Pre-module download assignment* to obtain all the necessary software and data.
+
+Create a new script named "ANOVA". As before, you begin by loading your packages with `library()`.
+```{r message=FALSE, warning=FALSE}
+library(tidyverse)
+```
+
+## Explore the metadata
+In addition to measurements of microbial communities, you also have geochemical data for the Saanich Inlet samples. For a brief introduction to these data, see Hallam SJ et al. 2017. Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone. Sci Data 4: 170158 [doi:10.1038/sdata.2017.158](https://www.nature.com/articles/sdata2017158). 
+
+A subset of these data has been provided in `4.MICB301_stats_extend_data.csv`  on Canvas including:
+
+  - Depth: depth in meters
+  - In micromolar (uM) or nanomolar (nM)
+	- NO3_uM: nitrate 
+	- NCTD_O2: nitrite
+	- N2O_uM: nitrous oxide
+	- NH4_uM: ammonium
+	- H2S_uM: hydrogen sulfide
+	- CTD_O2: oxygen
+	- CH4_uM: methane
+
+Read `4.MICB301_stats_extend_data.csv` into R using `read_csv` and save as `raw_dat`.
+```{r}
+raw_dat <- geochemicals
```

Comment:
I think it's confusing to tell students to save the csv file as raw_dat, but the code saves the object geochemicals to raw_dat instead